src/query/src/qAggMain.c

```diff
@@ -5186,7 +5186,7 @@ static bool elapsedSetup(SQLFunctionCtx *pCtx, SResultRowCellInfo* pResInfo) {
 }
 
 static int32_t elapsedRequired(SQLFunctionCtx *pCtx, STimeWindow* w, int32_t colId) {
```

Comment:
maybe we can remove elapsedRequired function and use more general dataBlockRequired function instead? Since after this change there's no difference of adding elapsed specified version of this function.